controllers/TodoController.js

```diff
@@ -0,0 +1,130 @@
+const sequelize = require('../config/db')
```

Comment:
bisa juga diambil dari connection di model ya
sehingga gak perlu buat connection db baru
```suggestion
const { sequelize } = require("../models");
```